runtime/onert/backend/cpu/ops/PadLayer.h

```diff
@@ -39,12 +39,10 @@ class PadLayer : public ::onert::exec::IFunction
   PadLayer();
 
 public:
-  void padFloat32();
-
-  void padQuant8();
+  template <typename T> void PadImpl(const T *constant_value_data);
```

Comment:
Please use small case for method's first spell.

Reply:
Oops.
I will fix it.